### PR TITLE
Create smart-commute-mntmo.nrel-op.json

### PR DESCRIPTION
**Pull request by sebastianbarry**
Created initial config file with details filled in from the MOU

### CODE REVIEW

File: configs/smart-commute-mntmo.nrel-op.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "server": {
+        "connectUrl": "https://smart-commute-mntmo-openpath.nrel.gov/api/",
+        "aggregate_call_auth": "user_only"
+    },
+    "intro": {
+        "program_or_study": "program",
+        "start_month": "2",
+        "start_year": "2023",
+        "mode_studied": "e-bike",
+        "program_admin_contact": "mike.salisbury@denvergov.org, 720 413-4271",
```
Comment (sebastianbarry) [inline configs/smart-commute-mntmo.nrel-op.json:13]:
```suggestion
        "program_admin_contact": "720.263.0106",
```

File: configs/smart-commute-mntmo.nrel-op.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                "why_we_collect_es": "Denver espera usar esta información para comprender mejor cómo se usan las bicicletas eléctricas, incluida la distancia recorrida, los viajes realizados, la reducción de emisiones asociada y los posibles impactos en la equidad.",
+                "why_we_collect" : "Smart Commute requiere que los participantes realicen un seguimiento de sus modos de viaje, el propósito y el modo que reemplazaron para todos los viajes y no solo para bicicletas eléctricas. Debido al componente de la primera y última milla bajo el contrato de Smart Commute con RTD, se les pide a los participantes que designen los diferentes modos que usan para cada viaje.",
```
Comment (sebastianbarry) [inline configs/smart-commute-mntmo.nrel-op.json:34]:
```suggestion
                "why_we_collect" : "Smart Commute requiere que los participantes realicen un seguimiento de sus modos de viaje, el propósito y el modo que reemplazaron para todos los viajes y no solo para bicicletas eléctricas. Debido al componente de la primera y última milla bajo el contrato de Smart Commute con RTD, se les pide a los participantes que designen los diferentes modos que usan para cada viaje.",
```

File: configs/smart-commute-mntmo.nrel-op.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                "why_we_collect_es": "Denver espera usar esta información para comprender mejor cómo se usan las bicicletas eléctricas, incluida la distancia recorrida, los viajes realizados, la reducción de emisiones asociada y los posibles impactos en la equidad.",
```
Comment (sebastianbarry) [inline configs/smart-commute-mntmo.nrel-op.json:33]:
```suggestion
```

File: configs/smart-commute-mntmo.nrel-op.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://smart-commute-mntmo-openpath.nrel.gov/api/",
```
Comment (sebastianbarry) [inline configs/smart-commute-mntmo.nrel-op.json:5]: @shankari can you please confirm which `connectUrl` we want for Smart Commute? Thanks!

Comment (shankari) [reply configs/smart-commute-mntmo.nrel-op.json:5]: The base URL is https://smart-commute-ebike-openpath.nrel.gov
Note that you also need to rename the config appropriately

Comment (sebastianbarry) [inline configs/smart-commute-mntmo.nrel-op.json:5]:
```suggestion
        "connectUrl": "https://smart-commute-ebike-openpath.nrel.gov/api/",
```

**Comment by shankari** [comment]
@sebastianbarry as I indicated in my comment, you need to rename the file as well.

>  Note that you also need to rename the config appropriately

File: configs/smart-commute-mntmo.nrel-op.json (hunk 4 above)
Comment (sebastianbarry) [reply configs/smart-commute-mntmo.nrel-op.json]: Resolved in https://github.com/e-mission/nrel-openpath-deploy-configs/pull/15/commits/fbe2cc78dbacd81d540201b1655299128200c0b7

**Pull request merged**